### PR TITLE
new technical contact check

### PR DESCRIPTION
**Pull request by owades**
# Description

Migrate existing technical contact check to v2.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation
- [ ] agencies.yml

## How has this been tested?

Verified pass and fail results.

### CODE REVIEW

File: warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__technical_contact_listed.sql [hunk 1]
```diff
@@ -0,0 +1,33 @@
+WITH feed_guideline_index AS (
+    SELECT * FROM {{ ref('int_gtfs_quality__schedule_feed_guideline_index') }}
+),
+
+feed_info AS (
+    SELECT
+        feed_key,
+        feed_contact_email,
+        EXTRACT(date FROM _valid_from) AS valid_from_date,
```
Comment (lauriemerrell) [inline warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__technical_contact_listed.sql:9]: As noted on #1969 I think you can drop the dates here

File: warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__technical_contact_listed.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        {{ technical_contact_listed() }} AS check,
+        {{ technical_contact_availability() }} AS feature,
+        CASE
+            WHEN LOGICAL_AND(t2.feed_contact_email IS NOT NULL) THEN "PASS"
+            ELSE "FAIL"
+        END AS status
+    FROM feed_guideline_index AS t1
+    LEFT JOIN feed_info AS t2
+       ON t1.feed_key = t2.feed_key
+      AND t1.date >= t2.valid_from_date
```
Comment (lauriemerrell) [inline warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__technical_contact_listed.sql:28]: As noted on https://github.com/cal-itp/data-infra/pull/1969 I think you can drop the dates here and just use feed key

**Comment by owades** [comment]
merged main, removed joins and ran tests. looks good!

**Review by lauriemerrell — APPROVED** [review]
LGTM! Also excited to see how short this PR is (with the YAML anchors & simpler joins).... Hopefully this v2 stuff is starting to seem like efficiency gains as you get acclimated!

**Review by lauriemerrell — APPROVED** [review]

File: warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__technical_contact_listed.sql [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        t1.date,
+        t1.feed_key,
+        {{ technical_contact_listed() }} AS check,
+        {{ technical_contact_availability() }} AS feature,
+        CASE
+            WHEN LOGICAL_AND(t2.feed_contact_email IS NOT NULL) THEN "PASS"
+            ELSE "FAIL"
+        END AS status
+    FROM feed_guideline_index AS t1
+    LEFT JOIN {{ ref('dim_feed_info') }} AS t2
```
[the remark this answers is not in the document]
Comment (lauriemerrell) [reply warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__technical_contact_listed.sql:17]: Sorry I was slow on this, but could maybe just tweak in whatever next PR you do on these?

[the remark this answers is not in the document]
Comment (owades) [reply warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__technical_contact_listed.sql:17]: too slow! I'll update on the next PR, incoming

**Pull request merged**